File: 4-Patron/php/TexturaSkin.php
<?php
class TexturaSkin
{
  private $rutaTextura;

  function __construct($rutaTextura)
  {
    $this->rutaTextura = $rutaTextura
  }

  function getRuta()
  {
    return $this->rutaTextura;
  }

  function setRuta($nRuta)
  {
    $this->rutaTextura = $nRuta;
  }

  function mostrarTextura()
  {
    echo "<img src='" . $rutaTextura . "' alt='skin'>"
  }
}
?>

File: 4-Patron/Java/TexturaSkin.java
package patron4;

public class TexturaSkin {
    private String RutaTextura;

    public TexturaSkin(String rutatextura) {
        this.RutaTextura = rutatextura;
    }

    public String getRutaTextura() {
        return RutaTextura;
    }

    public void setRutaTextura(String RutaTextura) {
        this.RutaTextura = RutaTextura;
    }
    
    
    public void mostrarTextura(){
        System.out.println("<img src = "+ this.RutaTextura + "alt = 'skin'>");
    }
}
